src/NHibernate.Test/NHSpecificTest/NH2302/Fixture.cs

```diff
@@ -152,6 +153,8 @@ public void BlobWithLength()
 				[Test]
 				public void BlobWithoutLength()
 				{
+					if (Dialect is MySQLDialect)
```

Comment:
We need to fix it for all dialects actually. Currently we have a huuuuuuuuge mess in how we handle string types.